homework-submission/App.js

```diff
@@ -0,0 +1,105 @@
+const path = '/data.json';
+
+const ListItem = ({ id, description, deadline, className }) => {
```

Comment:
Less components in react means less readability.

you only have two!

Reply:
New components are introduced.